testenv/gcp/asset/asset.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package asset
+
+import (
+	"context"
+	"fmt"
+	"github.com/openclarity/vmclarity/core/to"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	"cloud.google.com/go/compute/apiv1/computepb"
+)
+
+func Create(ctx context.Context, instancesClient *compute.InstancesClient, projectID string, zone string, assetName string) error {
+	var diskSizeGb int64 = 10
+
+	req := &computepb.InsertInstanceRequest{
```

Comment:
lets declare values used for instance creation as consts on top level (machineType, sourceImage, networkInterfaceName). do we need to use e2-standard-2 image for testing? can one of micro instances do the job?